code/game/atoms.dm

```diff
@@ -145,6 +149,10 @@
 
 	if (canSmoothWith)
 		canSmoothWith = typelist("canSmoothWith", canSmoothWith)
+	if(mapload && material)
+		if(!istype(material, /datum/material))
+			material = getmaterialref(material) //Get the proper instanced version

```

Comment:
You dont need to istype, it'll just be null if it isnt correct
`material = getmaterialref(material) || material`